colabs/huggingface/LLM_Finetuning_Notebook.ipynb

```diff
@@ -0,0 +1,437 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.            report_to="wandb",
</pre><p>I'd also make a direct mention that this is the flag to be set to get the one line integration working</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>